microservices/site-launch/step-function-workflows/site-launch.yaml

```diff
@@ -0,0 +1,2 @@
+name: SiteLaunch${self:service}${opt:stage, 'dev'}
+definition: ${file(./microservices/site-launch/step-function-workflows/site-launch.asl.yaml)}
```

Comment:
seems to be unused at present and points to the `site-launch.asl.yaml` so maybe we should just remove this?

Reply:
Sorry again for this - this is modified downstream!